### PR TITLE
Doc: about/trademark translated to pt-br

### PR DESCRIPTION
**Pull request by thalees**
In this PR was translated the page:
:large_blue_circle: **Medium priority** - 1. Page: About
- [x] trademark.md

@nodejs/nodejs-pt

### CODE REVIEW

File: locale/pt-br/about/trademark.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: about.hbs
+título: Política de Marca Registrada
+---
+# Política de Marca Registrada
+
+As marcas registradas do Node.js, marcas de serviço e marcas gráficas símbolos de
```
Comment (TiagoDanin) [inline locale/pt-br/about/trademark.md]: `... marcas gráficas são símbolos de...`

File: locale/pt-br/about/trademark.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+qualidade, desempenho e facilidade de uso com as quais as pessoas se associaram
+ao software e projeto do Node.js. Para garantir que as marcas do Node.js continuem
+simbolizar essas qualidades, devemos assegurar que as marcas sejam usadas apenas de formas
+que não enganaem as pessoas ou fazem com que confundam o Node.js com outro software
```
Comment (TiagoDanin) [inline locale/pt-br/about/trademark.md]: `enganaem` ~> `enganem`

**Review by TiagoDanin — CHANGES_REQUESTED** [review]

**Comment by TiagoDanin** [comment]
**Note:** Description of pull request and title must be in English

File: locale/pt-br/about/trademark.md (hunk 2 above)
Comment (thalees) [reply locale/pt-br/about/trademark.md]: Done.

File: locale/pt-br/about/trademark.md (hunk 1 above)
Comment (thalees) [reply locale/pt-br/about/trademark.md]: Done.

**Review by TiagoDanin — APPROVED** [review]
:100: LGTM

**Review by glaucia86 — APPROVED** [review]
LGTM 👍 

Thank you for your contribution @thalees

**Comment by glaucia86** [comment]
cc: @Maledong @Trott I approved it this translation pt-br

**Pull request merged**